gradle.properties

```diff
@@ -57,15 +57,15 @@ windowsProteomicsBinariesVersion=1.0
 # The current version numbers for the gradle plugins.
 artifactoryPluginVersion=4.21.0
 gradleNodePluginVersion=3.0.1
-gradlePluginsVersion=1.28.0
+gradlePluginsVersion=1.29.0-undeployModuleWarnings-SNAPSHOT
 owaspDependencyCheckPluginVersion=5.2.1
 versioningPluginVersion=1.1.0
 
 # Versions of node and npm to use during the build. If set, these versions
 # will be downloaded and used. If not set, the existing local installations will be use
 # The version of npm corresponds to the given version of node
-npmVersion=6.13.4
-nodeVersion=12.16.0
+npmVersion=6.14.13
+nodeVersion=14.17.1
```

Comment:
I'm not noticing anything on my local setup but this does put us in the "it will possibly break" land for a couple dependencies of `webpack-dev-server`, namely, `chokidar` and `fsevents`. 

![image](https://user-images.githubusercontent.com/3926239/123179410-33612f00-d43e-11eb-850c-83d14f7c7751.png)

Fortunately, they're [hard at work on v4](https://github.com/webpack/webpack-dev-server/releases) of `webpack-dev-server` which has much updated dependencies. This is something we can keep an eye on when working through package updates.